src/components/appBar/CyVerseAppBar.js

```diff
@@ -45,7 +44,7 @@ const useStyles = makeStyles((theme) => ({
         flexGrow: 1,
     },
     appBar: {
-        backgroundColor: palette.blue,
+        backgroundColor: theme.palette.blue,
```

Comment:
I don't think this will work, because I don't think the theme's `palette` object has `blue` defined (or `darkBlue`, or `white`, etc.) by default.

Can you verify if the theme colors are displaying correctly in storybook? Or are there any errors in the console?

We could probably add these fields to the `createMuiTheme` call in the index file, but I think it's better to stick with the theme's intentions where possible.
So instead of `theme.palette.blue`, I think this should be `theme.palette.primary`.
And instead of `theme.palette.red`, try `theme.palette.error`.
See https://material-ui.com/customization/default-theme/?expand-path=$.palette for the default fields in the `palette`.

Reply:
I made sure that there weren't any rendering or console errors in Storybook before I filed the PR. 

If I'm understanding things correctly, the ui-lib settings override or add to default MUI theme, so if something isn't overridden the MUI default will be used. I'll double/triple-check that that's the case, though.

Reply:
I ask, because in the index file, I only see a reference to ui-lib's `palette.blue` and `palette.lightBlue` in the `createMuiTheme` call, so I'm not sure how the MUI theme would be able to access all the other colors from the `ui-lib` palette.

Reply:
Okay, I've changed:
- `theme.palette.blue` references to `theme.palette.primary`
- `theme.palette.red` references to `theme.palette.error`
- ui-lib's palette is now merged into the default palette

That last one should hopefully pick up any missing color references.